framework/src/org/checkerframework/framework/type/SupertypeFinder.java

```diff
@@ -196,6 +197,18 @@
             return supertypes;
         }
 
+        private void subTypeVarsInWildcards(
+                Map<TypeParameterElement, AnnotatedTypeMirror> mapping) {
+            for (AnnotatedTypeMirror atm : mapping.values()) {
+                if (atm.getKind() == TypeKind.WILDCARD) {
+                    // Because the upper bound of the wildcard is copied from the upper bound of
+                    // the type parameter in #fixWildcardBound, it might contain references to
+                    // other type variables.  This replaces thos references.
```

Comment:
Maybe add why this is done here instead of as part of `fixWildcardBound`. Because you need the full mapping?